.pre-commit-config.yaml

```diff
@@ -31,6 +31,6 @@ repos:
           - --confidence-level=medium
           - --skip=B608
   - repo: https://github.com/syslabcom/slc.pre-commit
-    rev: main
+    rev: v1.0.0
```

Comment:
Sorry, can we please stick to consolidate version number schemes (i.e. 1.0.0)?
I have seen too much in my life to go for the version number you are proposing.